sbt-plugin/src/main/scala/kalix/sbt/KalixPlugin.scala

```diff
@@ -73,11 +74,11 @@ object KalixPlugin extends AutoPlugin {
         "io.kalix" %% "kalix-scala-sdk-protobuf-testkit" % KalixSdkVersion % Test),
       logConfig := "logback-dev-mode.xml",
       run / javaOptions ++=
-        dockerComposeUtils.value.readServicePortMappings
-        ++ Seq(
+        dockerComposeUtils.value.localServicePortMappings ++
```

Comment:
When we configure the locally run service, we use the service port mappings we read from the docker file, but remove any references to hosts. We only need the ports.